smartmin/static/js/heavy_data.js

```diff
@@ -1,252 +0,0 @@
-if (!window['django_select2']) {
```

Comment:
This file doesn't appear to be used anywhere - I think maybe it's just leftover from when smartmin did select2 stuff